src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -30,6 +30,34 @@ internal void VerifyDiagnostics(string source, CSharpCompilationOptions options,
     [CompilerTrait(CompilerFeature.LocalFunctions)]
     public class LocalFunctionTests : LocalFunctionsTestBase
     {
+        [ConditionalFact(typeof(DesktopOnly))]
```

Comment:
Should it be DesktopOnly? The test code does not look as possibly platform specific.

Reply:
In theory no it shouldn't be desktop only. This particular test hit a problem in our CoreClr infrastructure code though. To unblock this PR I had @t-camaia mark it as desktop only. Andy and I are discussing the bigger problem this exposed and how best to work around it.